.gitignore

```diff
@@ -1 +1,6 @@
 .DS_Store
+.idea/inspectionProfiles/profiles_settings.xml
```

Comment:
I think you can just specify the directory name with / at the end and that should cause git to ignore all contents in that subdirectory